src/commands.jl

```diff
@@ -213,6 +213,49 @@ function next_line!(@nospecialize(recurse), frame::Frame, istoplevel::Bool=false
 end
 next_line!(frame::Frame, istoplevel::Bool=false) = next_line!(finish_and_return!, frame, istoplevel)
 
+is_macro_expansion(lin::LineInfoNode) = lin.method == Symbol("macro expansion")
+is_macro_expansion(::LineNumberNode) = false
```

Comment:
Hmm, I might prefer this to be handled by `return isa(linfo, LineInfoNode) ? is_macro_expansion(linfo) : false` check in the method for `::Frame`. Wouldn't want this used on an expression that someone had manually `macroexpand`ed.